INSTALL.md

```diff
@@ -85,6 +85,7 @@ You can link OpenTelemetry C++ SDK with libraries provided in
    - `-DWITH_OTLP_GRPC=ON` : To enable building OTLP GRPC exporter.
    - `-DWITH_OTLP_HTTP=ON` : To enable building OTLP HTTP exporter.
    - `-DWITH_PROMETHEUS=ON` : To enable building prometheus exporter.
+   - [`-DOPENTELEMETRY_INSTALL=ON`](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/sdk/CMakeLists.txt): To install `libopentelemetry_common.so` needed for external code linking that rely on `ld`.
```

Comment:
This CMake option is not just to install libopentelemetry_common.so, but to install all the built otel-cpp libraries

Reply:
I think link to the source code is not needed here.

Reply:
Updated.
Removed code link, and updated description.